machine/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "machine"
```

Comment:
`ed-machine`

Reply:
I was wondering if we might reconsider the `Machine` type name also given that the FSM is already the "machine"... `EventDriver` or something?